drk-ontology.ttl

```diff
@@ -1,20 +1,339 @@
+    rdfs:comment "A property that specifies the name of a character in a creative work or during an event."@en .
+
+# The drk:characterType property specifies the type of a character in a creative work or during an event.
+# The character type values would be URIs from the DRK controlled vocabulary 'theatrical character types': `http://w3id.org/drk/vocabs/theatrical-character-types#`.
+# Examples of character types include person, animal, organization, puppet, robot, inanimate object, imaginary entity, etc.
+drk:characterType
+    a            rdf:Property, owl:ObjectProperty ;
+    rdfs:domain  drk:CharacterPerformanceRole ;
+    rdfs:range   schema:URL ;    # The value would be an URI from the DRK 'theatrical character types' vocabulary.
+    rdfs:label   "Character Type"@en ;
+    rdfs:comment "A property that specifies the type of a character in a creative work or during an event."@en .
+
+# The drk:ProductionRole class links a production role to its performer.
+# Production roles can, in general, include creator roles, non-character roles, offstage or behind-the-scenes roles.
+# Examples of production roles include creator, writer, playwright, costume designer, stage designer, choreographer, makeup artist, understudy, usher, funder, sponsor, etc.
+drk:ProductionRole
+    a               owl:Class ;
+    rdfs:subClassOf drk:Role ;
+    rdfs:label      "Production Role"@en ;
+    rdfs:comment    "A class that links a production role to its performer. Production roles can, in general, include creator roles, non-character roles, offstage or behind-the-scenes roles. Examples of production roles include creator, writer, playwright, costume designer, stage designer, choreographer, makeup artist, understudy, usher, funder, sponsor, etc."@en .
+
+# The drk:PlayProduction class includes every stage of Play Production from writing, planning, and casting, to rehearsal, except for the presentation of work, i.e., Event.
+# Event is modelled using another class: drk:TheaterEvent.
+# It extends schema:CreativeWork class.
+drk:PlayProduction
+    a               owl:Class ;
+    rdfs:subClassOf schema:CreativeWork ;
+    rdfs:label      "Play Production"@en ;
+    rdfs:comment    "A class that includes all stages of Play Production from writing, planning, casting, and staging to rehearsal, but excludes the final presentation or performance, which is modeled as an Event using the drk:TheaterEvent class."@en .
+
+# The drk:productionType property specifies the production type of a creative work, such as 'World Premiere', 'Language Premiere', etc.
+# The production type values would be URIs from the DRK controlled vocabulary 'theatrical production types': `http://w3id.org/drk/vocabs/theatrical-production-types#`.
+drk:productionType
+    a           rdf:Property, owl:ObjectProperty ;
+    rdfs:domain drk:PlayProduction ;
+    rdfs:range  schema:URL ;    # The value would be an URI from the DRK 'theatrical production types' vocabulary.
```

Comment:
I was a bit skeptical about using `skos:Concept` as range since not all vocabularies are defined as `skos:ConceptScheme`s, for example the [OMG's countries vocabulary](https://www.omg.org/spec/LCC/20211101/Countries/ISO3166-1-CountryCodes.ttl).
However, at least for the cases where we have created new vocabularies in the DRK namespace, we can definitely set the range as `skos:Concept`.
This is done!